units/4DC/UAL0204/UAL0204_unit.bp

```diff
@@ -54,7 +54,7 @@ UnitBlueprint {
 
             MuzzleVelocity = 60,
 
-            ProjectileId = "/mods/4DC/projectiles/Sniper_Piercing/Sniper_Piercing_proj.bp",
+            ProjectileId = "/mods/loud-community-edition/projectiles/sniper_nonpiercing/sniper_nonpiercing_proj.bp",
```

Comment:
Why did you make this lowercase? Is it required because in my change for atleast the BP paths. It does not matter if its lowercase.

Reply:
Didn't resolve for me (blank projectile).  I can 2x check.

Reply:
![image_2024-09-08_094401138](https://github.com/user-attachments/assets/4fc30fdc-77a2-488e-ba21-e7d459b4511f)

yeah i just checked my commit, yeah i had it capitalized strange that it doesnt work for you?

Reply:
Mine works both ways.

Reply:
I think projectileID gets downcased either way, probably why it works.  I would prefer to have it reflect the actual file path though, so changing it.